<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLineDialog</class>
 <widget class="QDialog" name="EditLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Line</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="componentLabel">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Component</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="highlightColorLineEdit">
     <property name="text">
      <string>#ff000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="borderWidthSpinBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="lineColorButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../scwx-qt.qrc">
       <normaloff>:/res/icons/font-awesome-6/palette-solid.svg</normaloff>:/res/icons/font-awesome-6/palette-solid.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="borderColorButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../scwx-qt.qrc">
       <normaloff>:/res/icons/font-awesome-6/palette-solid.svg</normaloff>:/res/icons/font-awesome-6/palette-solid.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="borderLabel">
     <property name="text">
      <string>Border</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineColorLineEdit">
     <property name="text">
      <string>#ff000000</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lineLabel">
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::Reset</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="lineColorFrame">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="highlightWidthSpinBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="highlightColorFrame">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="colorLabel">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="borderColorLineEdit">
     <property name="text">
      <string>#ff000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="borderColorFrame">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="widthLabel">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="lineWidthSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="highlightLabel">
     <property name="text">
      <string>Highlight</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="highlightColorButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../scwx-qt.qrc">
       <normaloff>:/res/icons/font-awesome-6/palette-solid.svg</normaloff>:/res/icons/font-awesome-6/palette-solid.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QWidget" name="lineLabelContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../scwx-qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditLineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditLineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
